### PR TITLE
WIP: Fix norm event for multiplicity interval

### PR DESCRIPTION
**Pull request by lvermunt**
- [ ] Fix to be checked
- [ ] Part2 of the fix to be discovered/difference to be understood (seeing less events as AliNormalizationCounter)

### CODE REVIEW

File: machine_learning_hep/analysis/analyzerdhadrons_mult.py [hunk 1]
```diff
@@ -507,12 +511,19 @@ def makenormyields(self):
             lfile = TFile.Open(self.n_filemass)
             hNorm = lfile.Get("hEvForNorm_mult%d" % imult)
             normfromhisto = hNorm.GetBinContent(1)
+
+            lvar2_binmincorr = self.lvar2_binmin[imult]
+            lvar2_binmaxcorr = self.lvar2_binmax[imult]
+            if len(self.p_nbin2) > 2:
+                if self.lvar2_binmax[1] == self.lvar2_binmin[2]:
```
Comment (vkucera) [inline machine_learning_hep/analysis/analyzerdhadrons_mult.py]: This doesn't look right.
Should it be something like this?
`if imult < self.p_nbin2 - 1:`
`    if self.lvar2_binmax[imult] == self.lvar2_binmin[imult + 1]:`

Comment (lvermunt) [reply machine_learning_hep/analysis/analyzerdhadrons_mult.py]: Hi Vit, thanks! I opened this PR quickly without testing it yet. Should be indeed like you wrote. @ginnocen, please note you need this fix when you continue to debug.

**Comment by ginnocen** [comment]
Hi @LuukVermunt @vkucera thanks to both. I will fix it in the current PR where, as discussed, I merged this validation step in the mass histo creation. So far I reproduce the old results with a simpler code. I will start soon, or maybe tomorrow, to check differences wrt to the task. 

Please provide feedback if you have time. 
https://github.com/ginnocen/MachineLearningHEP/pull/585

**Comment by ginnocen** [comment]
Superseded by #585

**Pull request closed without merging**